.github/workflows/go-tests.yml

```diff
@@ -22,7 +22,6 @@ permissions:
 env:
```

Comment:
No updates needed to triggers as they were already split on PR and protected branch push.